x/vendorinfo/genesis_test.go

```diff
@@ -3,12 +3,39 @@ package vendorinfo_test
 import (
 	"testing"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 	keepertest "github.com/zigbee-alliance/distributed-compliance-ledger/testutil/keeper"
+	dclauthtypes "github.com/zigbee-alliance/distributed-compliance-ledger/x/dclauth/types"
 	"github.com/zigbee-alliance/distributed-compliance-ledger/x/vendorinfo"
 	"github.com/zigbee-alliance/distributed-compliance-ledger/x/vendorinfo/types"
 )
 
+type DclauthKeeperMock struct {
```

Comment:
Can we define such mock type within `vendorinfo` cosmos module only once?

Reply:
fixed